apps/central/src/components/Groups.vue

```diff
@@ -98,6 +98,7 @@
         @close="closeEditSchema"
         :schemaName="showEditSchema"
         :schemaDescription="editDescription"
+        :schemaIsChangelogEnabled="editIsChangelogEnabled"
```

Comment:
can we use our 'settings' component for managing these settings instead of programming them?

Reply:
BTW: I think it is helpful to somewhere program these settings (instead of keeping them totally free text) so user knows what settings we actually do support